linkerd/app/core/src/errors.rs

```diff
@@ -36,12 +36,12 @@ pub enum Reason {
     Unexpected,
 }
 
-#[derive(Debug)]
-pub struct NewRespond<B>(std::marker::PhantomData<fn() -> B>);
+#[derive(Copy, Clone, Debug)]
+pub struct NewRespond(());
```

Comment:
note: this is done to prevent instantiation `NewRespond` directly from outside of this module. not critical but something we do frequently to encourage going through the constructors.